3DAmsterdam/Assets/Amsterdam3D/Scripts/Interface/WarningDialogs.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class WarningDialogs : MonoBehaviour
+{
+    public static WarningDialogs Instance;
+
+    [SerializeField]
+    private Warning warningPrefab;
+
+    private void Awake()
+    {
+        Instance = this;
+    }
+
+    /// <summary>
+    /// Spawns a new warning prefab with a message. Leave blank for default server warning.
+    /// </summary>
+    /// <param name="message">Optional custom message</param>
+    /// <param name="clearOldWarnings">Deletes all existing warnings</param>
+    public void ShowNewDialog(string message = "", bool clearOldWarnings = true)
+    {
+        if(clearOldWarnings)
+        {
+            foreach (Transform child in transform)
+                Destroy(child.gameObject);
```

Comment:
Waarom word hier telkens een nieuw gameobject aangemaakt ipv een gameobject die hergebruikt word?

Reply:
Dan kan je optioneel meerdere meldingen weergeven / opstapelen.  Bijv. wanneer een scene met een ID niet is gevonden, en een API er uit ligt wil je evt. allebij die meldingen kunnen tonen